rfcs/0000-rlp-status.md

```diff
@@ -0,0 +1,162 @@
+| Created    | True   | "PolicyCreated"           | "KuadrantPolicy created"                                                    | No        |
+| Updated    | True   | "PolicyUpdated"           | "KuadrantPolicy has been updated"                                           | No        |
+| Applied    | True   | "PolicyApplied"           | "KuadrantPolicy has been successfully applied                               | Yes       |
+|            | False  | "PolicyNotApplied"        | "KuadrantPolicy is invalid"                                                 |           |
+| Reconciled | True   | "PolicyReconciled"        | "KuadrantPolicy has been successfully reconciled"                           | Yes       |
+|            | False  | "PolicyNotReconciled"     | "KuadrantPolicy failed reconciliation"                                      |           |
+| Enforced   | True   | "PolicyEnforced"          | "KuadrantPolicy has been successfully enforced"                             | Yes       |
+|            | False  | "PolicyPartiallyEnforced" | "KuadrantPolicy has encountered some issues and has been partially applied" |           |
+|            | False  | "PolicyNotEnforced"       | "KuadrantPolicy has encountered an error and can't be applied"              |           |
+
+
+A simplified version and more aligned with the Kubernetes objects implementation could be represented as the following.
+
+**Conditions**
+
+All conditions are top-level.
+
+| Type        | Status | Reason                      | Message                                                                     |
+|-------------|--------|-----------------------------|-----------------------------------------------------------------------------|
+| Progressing | True   | "PolicyCreated"             | "KuadrantPolicy created"                                                    |
+|             | True   | "PolicyUpdated"             | "KuadrantPolicy has been updated"                                           |
+|             | True   | "PolicyApplied"             | "KuadrantPolicy has been successfully applied                               |
+|             | True   | "PolicyReconciled"          | "KuadrantPolicy has been successfully reconciled"                           |
+|             | False  | "PolicyEnforced"            | "KuadrantPolicy has been successfully enforced"                             |
+|             | False  | "PolicyError"               | "KuadrantPolicy has encountered an error"                                   |
+| Enforced    | True   | "PolicyEnforced"            | "KuadrantPolicy has been successfully enforced"                             |
+|             | False  | "PolicyPartiallyEnforced"   | "KuadrantPolicy has encountered some issues and has been partially applied" |
+|             | False  | "PolicyNotEnforced"         | "KuadrantPolicy has encountered an error and can't be applied"              |
```

Comment:
Is there an overlap between `Enforced: False (PolicyNotEnforced)` and `Failed: True (PolicyReconciliationError)` maybe?

What about specifying a more specific reason why the policy is not enforced? E.g.

```suggestion
|             | False  | "PolicyOverridden"         | "KuadrantPolicy overridden by KuadrantPolicy policy-ns/policy-name"              |
```